settings/Core.setting.php

```diff
@@ -976,6 +976,48 @@
     'description' => ts('If enabled, then CMS email id will be synchronised with CiviCRM contacts\'s primary email.'),
     'help_text' => NULL,
   ),
+  'theme_frontend' => array(
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'theme_frontend',
+    'type' => 'String',
+    'quick_form_type' => 'Select',
+    'html_type' => 'Select',
+    'html_attributes' => array(
+      'class' => 'crm-select2',
+    ),
+    'pseudoconstant' => array(
+      'callback' => 'call://themes/getAvailable',
+    ),
+    'default' => 'default',
+    'add' => '4.7',
+    'title' => 'Frontend Theme',
+    'is_domain' => 1,
+    'is_contact' => 0,
+    'description' => 'Theme to use on frontend pages',
+    'help_text' => NULL,
+  ),
+  'theme_backend' => array(
+    'group_name' => 'CiviCRM Preferences',
+    'group' => 'core',
+    'name' => 'theme_backend',
+    'type' => 'String',
+    'quick_form_type' => 'Select',
+    'html_type' => 'Select',
+    'html_attributes' => array(
+      'class' => 'crm-select2',
+    ),
+    'pseudoconstant' => array(
+      'callback' => 'call://themes/getAvailable',
+    ),
+    'default' => 'default',
+    'add' => '4.7',
+    'title' => 'Backend Theme',
```

Comment:
Matt has been adding ts() around these in the settings file, which makes sense

Reply:
Updated